kubernetes/deploy/01_metallb.yaml

```diff
@@ -0,0 +1,307 @@
+---
```

Comment:
Same with this file. I would probably put it in an opt folder and link to this manifest

Reply:
This is required to be able to use a service as `LoadBalancer`, I'm not sure that using an external load balancer should be useful for development purpose (and AFAIK Docker desktop do not support `LoadBalancer` service out of the box).
Do not enable it by default will require extra steps to contributors and only few of them will need to hack the default config.. @dannav WDYT?

Reply:
Yep, you're correct if you want to use a lb :-P. But you can do this with ingress if you want to test locally as well. 

Here's an example w/ minikube and nginx: https://kubernetes.io/docs/tasks/access-application-cluster/ingress-minikube/

I guess the point I'm trying to get across is that unfortunately there are a million ways to do these things :-(. If we want to show one way to do it that's fine. But if we do, we should be explicit that it is optional unless there's a really good reason why it should be done a specific way.